dandi/publish/tests/test_asset.py

```diff
@@ -73,3 +75,50 @@ def test_asset_rest_retrieve(api_client, asset):
         'updated': TIMESTAMP_RE,
         'metadata': asset.metadata,
     }
+
+
+@pytest.mark.django_db
+def test_asset_path(api_client, asset):
+    path = os.path.split(asset.path)
+    asset_directory = path[: len(path) - 1]
+    asset_filename = path[len(path) - 1]
```

Comment:
`os.path.split` always returns two strings, so you don't have to do this logic.
```suggestion
    asset_directory = path[0]
    asset_filename = path[1]
```